src/styles/components/_tile.scss

```diff
@@ -7,18 +7,9 @@
     padding-top: ($height / $width) * 100%;
     content: "";
   }
-
-  .mc-tile__content {
```

Comment:
Unnecessary to belong in the mixin as the variables assigned above have no bearing on these values.